Source/src_android/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        google()
+    }
+    dependencies {
+        classpath("com.android.tools.build:gradle:7.2.1")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${Constants.kotlinVersion}")
+        // releasing
+        classpath("com.github.dcendents:android-maven-gradle-plugin:2.1")
```

Comment:
Does this really work? This plugin has been long time abandoned.
There was a ton of issues regarding compatibility with recent Gradle and AGP versions.

Reply:
Yop, I copied it from an old project (but it still works though). Moved to the nexus publish plugin.